CorsixTH/Lua/hospital.lua

```diff
@@ -78,11 +80,12 @@ function Hospital:Hospital(world, avail_rooms, name)
   self.concurrent_epidemic_limit = level_config.gbv.EpidemicConcurrentLimit or 1
 
   -- Initial values
-  self.interest_rate = interest_rate
+  self.interest_rate = interest_rate / 10000
   self.inflation_rate = 0.045
+  self.overdraft_interest_rate = self.interest_rate + overdraft_differential / 10000
   self.salary_incr = level_config.gbv.ScoreMaxInc or 300
   self.sal_min = level_config.gbv.ScoreMaxInc / 6 or 50
-  self.reputation = 500
+  self.reputation = reputation
   self.reputation_min = 0
   self.reputation_max = 1000
```

Comment:
Shouldn't `self.reputation` be clamped against the stated bounds?
Likely stuff goes wrong when the reputation is outside expected bounds.